proxy/src/proxy.rs

```diff
@@ -1036,14 +1034,9 @@ impl<S: AsyncRead + AsyncWrite + Unpin> Client<'_, S> {
     }
 }
 
-pub fn neon_options(params: &StartupMessageParams) -> Option<String> {
+pub fn neon_options(params: &StartupMessageParams) -> Option<Vec<(String, String)>> {
```

Comment:
If empty `options` is considered none, can we remove the option? Feels unnecessary